### ISSUE TITLE
CycleFromList( [1,2,3,4] ) fails

### ISSUE THREAD

**Issue by huberfe** [gap-system/gap#3594]
CycleFromList( [1,2,3,4] ) fails.

[GAP manual Chpt 42](https://www.gap-system.org/Manuals/doc/ref/chap42.html), 42.5-5 CycleFromList

### Observed behaviour

`CycleFromList( [1,2,3,4] );`
`Error, Variable: 'CycleFromList' must have a value
not in any function at line 1 of *stdin*`

### Expected behaviour

`CycleFromList([1,2,3,4] );`
`(1,2,3,4)`

### Copy and paste GAP banner (to tell us about your setup)
```
┌───────┐   GAP 4.8.8, 20-Aug-2017, build of 2018-02-19 20:02:33 (UTC)
 │  GAP  │   https://www.gap-system.org
 └───────┘   Architecture: x86_64-linux-gnu-gcc-default64
 Libs used:  gmp, readline
 Loading the library and packages ...
 Components: small 2.1, small2 2.0, id2 3.0, trans 1.0, prim 2.1
 Packages:   AClib 1.3.1, Alnuth 3.1.1, AtlasRep 1.5.1, AutPGrp 1.10, CRISP 1.4.4, Cryst 4.1.19, CrystCat 1.1.9, CTblLib 1.2.2, FactInt 1.6.2, 
             FGA 1.4.0, GAPDoc 1.6.2, IRREDSOL 1.4, Polenta 1.3.8, Polycyclic 2.14, RadiRoot 2.8, Sophus 1.24, SpinSym 1.5.1, TomLib 1.2.8
```

**Comment by fingolfin** [comment]
`CycleFromList` is not available in GAP 4.8, it was only introduced in GAP 4.10. There are many other reasons why I'd recommend updating to GAP 4.10.2.

If you absolutely can't update, you could emulate the functionality via `PermList`; or extract the code for `CycleFromList` from https://github.com/gap-system/gap/pull/2242

**Comment by huberfe** [comment]
Thanks. That's was a bit unexpected, because I've just installed a new version this week. Seems there still was an old version installed by a package manager lurking around..
